data/tools/wesnoth/campaignserver_client.py

```diff
@@ -27,6 +27,7 @@ def append_tag(tag : wmlparser.TagNode, sub : str) -> wmlparser.TagNode:
 class CampaignClient:
     # First port listed will be used as default.
     portmap = (
+        ("15015", "1.15.x"),
```

Comment:
Is it OK to also backport this line to 1.14?

The goal is to use wesnoth_addon_manager from stable (1.14) branch to upload things to 1.15, when needed. As opposed to always using wesnoth_addon_manager from 1.15 to upload both the 1.14 and 1.15 version.

Reply:
Or I guess it doesn't make any practical sense anymore as 1.14 has around 0-1 releases left?

Reply:
The proble is that I'm doing it in a script. So I have to either introduce the same logic (port mapping) inside it, or make use of auto-detection from wesnoth_addon_manager.

I've made a PR: https://github.com/wesnoth/wesnoth/pull/4383